packages/near-fast-auth-signer/.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 
```

Comment:
I'm not able to create an account from Safari, I'm getting the following error: http://localhost:3002/?reason=Firebase%3A+Error+%28auth%2Fnetwork-request-failed%29.


https://github.com/near/fast-auth-signer/assets/22358004/a4d36070-fda9-4166-9530-d7fcbed33eb4